### DISCUSSION TITLE
Making BPM timing easier

### DISCUSSION THREAD

**Discussion by WinterIsPretty** [ppy/osu#15153] [Ideas]
Very big request, but I wish BPM timing would be visual like osu stable is

I wish you could move the beat lines to change the bpm automatically to make it so you could drag a beat to your cursor where you hear the next beat. This will make it extremely easy to do variable bpm songs and will save a lot of time. Also I saw the editor timing section in lazer and I think it is more confusing than the stable editor.

**Comment by integerrr** [comment]
a scrollable offset input box will probably feel better and more accurate than dragging a beat line to where you think the beat is?

**Comment by peppy** [comment]
Initial timing process is not yet implemented. You can be sure it will be more approachable than stable once it is.

**Comment by WinterIsPretty** [comment]
In stable it is super annoying to use the scroll to go where you think the beat is and it is way faster to just drag it immediately over
